okhttp/src/jvmTest/java/okhttp3/WebPlatformToAsciiData.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
```

Comment:
Block?

Reply:
Oooh, I copy-pasted and changed the date only. Fixed.